challenges/functions.js

```diff
@@ -7,23 +7,39 @@
   * The consume function should return the invocation of cb, passing a and b into cb as arguments
 */
 
+function consume(a, b, cb){
+  return cb(a, b);
+}
 
 /* Step 2: Create several functions to callback with consume();
   * Create a function named add that returns the sum of two numbers
   * Create a function named multiply that returns the product of two numbers 
   * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
 */
+function add(x, y, cb) {
```

Comment:
no need to add cb as parameter here because it doesn't do anything inside the function

Reply:
Sounds good. I thought the step 1 question was a little unclear... maybe I can ask you about it on Monday.

Reply:
yup, send me a DM in Slack tomorrow @andremichalowski